health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/appConfigurationRedesign/NavigationLogicWrapper.js

```
@@ -485,23 +562,90 @@ function NavigationLogicWrapper({
+                    height: "fit-content",
+                    padding: "0.25rem 0.5rem",
+                    width: "100%",
+                    maxWidth: "100%",
+                }}
+                className={"version-tag"}
+                labelStyle={{ whiteSpace: "normal", wordBreak: "break-word" }}
+            />
+
+            <div style={{ display: "inline-flex", alignItems: "center", gap: "0.75rem", whiteSpace: "nowrap" }}>
+                <div
+                    role="button"
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<button>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)